tap_stripe/__init__.py

```diff
@@ -106,6 +107,9 @@ class Context():
     stream_map = {}
     new_counts = {}
     updated_counts = {}
+    window_size = DEFAULT_DATE_WINDOW_SIZE # By default fetch data from last 30 days for newly created records.
+    event_update_window_size = DEFAULT_EVENT_UPDATE_DATE_WINDOW_SIZE # By default collect data of 7 days in one API
+                                                       # call for event_updates
```

Comment:
This won't look good, I guess it will be better to take an exception or change the comment place.